### PR TITLE
Use the value for app label instead of hardcode

### PR DESCRIPTION
**Pull request by hwchiu**
onos-tost helm chart needs to refer to the `app_label` in the network config pusher. Making this a helm value allows us to refer to it without duplicating.

### CODE REVIEW

**Review by charlesmcchan — APPROVED** [review]

**Review by Andrea-Campanella — COMMENTED** [review]
@hwchiu @charlesmcchan thanks for the patch. A favour I'm asking in the future, please wait for a review from me and/or @teone on the patch, both of our teams are using this chart and I'd like to avoid breaking each other's deployments or having to fix the version and not having the capability to move. Thanks!

**Comment by charlesmcchan** [comment]
@Andrea-Campanella This is part of a fix to an issue that's blocking the deployment of two Pronto production sites and is a relatively trivial / low risk change based on my judgement so I went ahead. Will make sure changes go through you or Teo in the future.

**Comment by teone** [comment]
@charlesmcchan no problem, if there are no external API changes and you need a quick turnaround is not an issue to get things merged.

@Andrea-Campanella we also depend on a fixed version of the chart, so things won't break under the hood for us.

**Pull request merged**
